oxidauth-kernel/src/authorities/mod.rs

```diff
@@ -31,11 +31,18 @@ pub struct Authority {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct AuthoritySettings {
     pub jwt_ttl: Duration,
+    pub jwt_nbf_offset: NbfOffset,
     pub refresh_token_ttl: Duration,
     pub totp: TotpSettings,
     pub entitlements_encoding: EntitlementsEncoding,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub enum NbfOffset {
+    Enabled(Duration),
```

Comment:
I feel like I almost get it, but could use an explanation for why this code works with `if let`...   but not the original `Enabled {offset: Duration}` syntax. With the original code, I kept getting an error like "Something which is neither a tuple struct nor a tuple variant was used as a pattern."